v4-client-py/v4_client_py/chain/aerial/client/utils.py

```diff
@@ -28,6 +28,8 @@ def prepare_and_broadcast_basic_transaction(
     :param account: The account
     :param gas_limit: The gas limit
     :param memo: Transaction memo, defaults to None
+    :param broadcast_mode: Broadcast mode, defaults to None
+    :param fee: Transaction fee, defaults to 5000
```

Comment:
nit: Note that the denomination is from the network config.

Reply:
done